src/angular-sql-query.js

```diff
@@ -572,9 +570,9 @@
    */
   function extractValues(filtersParameters = {}) {
     return Object.keys(filtersParameters)
-      .map(key => filtersParameters[key])
+      .map(key => filterValuesToSQLBindingsValues(filtersParameters[key]))
       .reduce((acc, value) => acc.concat(value), []); // flatten array values 
-    // extractValues({} a: 1, b: [2, 3] }) => [1, 2, 3]
+    // extractValues({} a: 1, b: [2, 3], c: /toto/ }) => [1, 2, 3, '%toto%']
```

Comment:
Can we move this sample in jsdoc ?

Reply:
![image](https://user-images.githubusercontent.com/1308790/75434607-1c95ef80-5952-11ea-972e-6f91904716e5.png)
Its looking really nice in editor ;)

Reply:
While I often brag against jsdoc I think it's a good tool for libs 👍